### ISSUE TITLE
Rune Arc

### ISSUE THREAD

**Issue by GexoGold** [Ariescyn/EldenRing-Save-Manager#58]
Hello!
Can`t add more Rune Arcs with Inventory Editor. It`s almost okey, when i`m trying to set them in the programm, but in game inventory they don`t appear.

**Comment by Ariescyn** [comment]
Yeah rune arcs can be weird since they can have two variants. Try using the item replacer to replace a consumable with rune arc and then try to set the quantity

**Comment by GexoGold** [comment]
Unfortunately, i dont understand, how does this function work... Please, explain in a few steps. Thanks!
